src/deconvolution.jl

```diff
@@ -53,20 +53,20 @@ julia> img_n = poisson(img_b, 300);
 julia> res, o = deconvolution(img_n, psf);
 ```
 """
-function deconvolution(measured::AbstractArray{T, N}, psf;
-        loss=Poisson(),
-        regularizer=GR(),
-        λ=T(0.05),
-        background=zero(T),
-        mapping=Non_negative(),
-        iterations=nothing,
-        conv_dims = ntuple(+, ndims(psf)), 
-        padding=0.00,
-        opt_options=nothing,
-        opt=LBFGS(linesearch=BackTracking()),
-        initial=mean(measured),
-        debug_f=nothing,
-        opt_package=Opt_Optim) where {T, N}
+function deconvolution(measured::AbstractArray{T,N}, psf;
+    loss=Poisson(),
+    regularizer=GR(),
+    λ=T(0.05),
+    background=zero(T),
+    mapping=Non_negative(),
+    iterations=nothing,
+    conv_dims=ntuple(+, ndims(psf)),
+    padding=0.00,
+    opt_options=nothing,
+    opt=LBFGS(linesearch=BackTracking()),
+    initial=mean(measured),
+    debug_f=nothing,
+    opt_package=Opt_Optim) where {T,N}
```

Comment:
can you maybe revert this again?

Reply:
I truly do not know, how is this happening... I must have some kind of hook set up pre-push because I do not have these locally.. Can you think of a reason, why this keeps happening? I will revert those.

Reply:
Isn't that your editor?
Maybe your master branch is not synced with upstream anymore

Reply:
I think this was because I had the wrong repo set as the default one. I created a new PR.